x/group/types_test.go

```diff
@@ -395,14 +395,14 @@ func TestGroupAccountInfo(t *testing.T) {
 			timeout:      proto.Duration{Seconds: 1},
 			expErr:       true,
 		},
-		"invalid group account address": {
+		"valid variable length group account address": {
 			group:        1,
 			groupAccount: []byte("any-invalid-group-address"),
```

Comment:
maybe change the string here too? (removing "invalid" which makes it a bit confusing now)